src/systemctl/systemctl.c

```diff
@@ -6001,6 +6001,24 @@ static int mangle_names(char **original_names, char ***mangled_names) {
         return 0;
 }
 
+static int normalize_filenames(char **names, bool warn_if_path) {
```

Comment:
because i'm stupid...

I started by copy/pasting normalize_name and missed the unused parameter. fixing...